### ISSUE TITLE
app-emulation/vmware-workstation: Tools that aren't macOS/Darwin don't install

### ISSUE THREAD

**Issue by shadowgamer67890** [stefantalpalaru/gentoo-overlay#151]
This isn't about the "Install VMware Tools..." button not working, I know you need to manually pick the iso from /opt/vmware/lib/vmware/isoimages. Thing is that directory has been barren for the last update or two aside from the manifest txt file, and enabling every single guest tool USE flag interestingly only results in macOS tools appearing. Going and extracting the tarballs from the softwareupdate.vmware.com site isn't too much of a hassle, but the Windows ones were definitely there when I first installed the package and now they don't seem to be...

emerge info, build log, and pqv just because
[info.txt](https://github.com/user-attachments/files/16447747/info.txt)
[build.log](https://github.com/user-attachments/files/16447746/build.log)
[pqv.txt](https://github.com/user-attachments/files/16447748/pqv.txt)

**Comment by stefantalpalaru** [comment]
> Going and extracting the tarballs from the softwareupdate.vmware.com site isn't too much of a hassle

Which tarball and where exactly is the Windows ISO?

**Comment by shadowgamer67890** [comment]
They can be found at
https://softwareupdate.vmware.com/cds/vmw-desktop/ws/17.5.2/23775571/linux/packages/

Each one is a .component file with a descriptor.xml inside of a tarball, and the .iso lives inside of the .component. I have no idea how the .component file is compressed but I'm able to get the .iso out of it with 7-Zip so that's how I've been getting it. Hitting the info button while in 7-Zip claims gzip compression, but I can't actually get it to unzip from the command line that way, so I'm not sure what's going on there.

**Comment by shadowgamer67890** [comment]
So first I've realized my earlier comment on the version in the metadata.xml is totally unrelated to the vmware-installer version so ignore that part, but I've figured out enough about the component files to get the iso out without 7-Zip. They seem to consist of:

-Some unknown header
-The exact data of the descriptor.xml also found in the tarball
-The ISO gzipped

all back to back. This ugly script got the ISO out by way of grepping the offset of `<component/>` which every descriptor.xml has, adding 12 which is the byte length of said component end tag, lopping off the header and xml, and finally gunzipping the iso.

`tar -xf vmware-tools-windows-12.4.0-23259341.x86_64.component.tar`
`TAGOFF=$(grep -boa "</component>" vmware-tools-windows-12.4.0-23259341.x86_64.component | cut -d ":" -f 1)`
`dd if=vmware-tools-windows-12.4.0-23259341.x86_64.component of=windows.iso.gz bs=$(($TAGOFF+12)) skip=1`
`gzip -d windows.iso.gz`

Perhaps this can be incorporated into the ebuild in some way?
